### DISCUSSION TITLE
Is it possible to define partitions in the stm32 backup sram ?

### DISCUSSION THREAD

**Discussion by joelguittet** [zephyrproject-rtos/zephyr#63098] [Q&A] [answered]
Hello community,

I wonder if it is possible to define a partition in the backup sram. I'm using an STM32H745 MCU. Goal is to use it for the storage partition (nvs).

I have tried to add the following:

```
&backup_sram{
    partitions {
        compatible = "fixed-partitions";
        #address-cells = <1>;
        #size-cells = <1>;
        /* Storage slot: 4KB */
        storage_partition: partition@38800000 {
            label = "storage";
            reg = <0x38800000 DT_SIZE_K(4)>;
        };
    };
    status = "okay";
};
```

But it seems nothing is generated, except this is enabling the backup sram.

Joel

**Comment by nordicjm** [comment, marked as answer]
No

**Reply by joelguittet** [reply]
Clear and short! Ok thanks for the answer!
